.github/workflows/containerize-invehicle-stack-chariott.yml

```diff
@@ -1,4 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
 # SPDX-License-Identifier: MIT
+

```

Comment:
Should the version be incremented when the files are changed?

Eclipse Symphony has a release workflow (https://github.com/eclipse-symphony/symphony/blob/main/.github/workflows/release.yaml#L121) that increments their image versions.

This is the workflow step that does the incrementing https://github.com/eclipse-symphony/symphony/blob/main/.github/workflows/release.yaml#L62

Reply:
Thanks for the information and I do like the suggestion. I will consider using a similar approach when we introduce releases/ release pipelines into the blueprint-related work. However, for right now I would like to keep the version as-is until we have at least one stable version of each image contributed.

Another note is that the version of these files will match the version specified in the original repository. I will look into how we can ensure this programmatically in a later change.